src/Python/tests/tests.py

```diff
@@ -17,41 +18,49 @@
 from wcon import WCONWorms, MeasurementUnit
 from wcon.measurement_unit import MeasurementUnitAtom
 
+
 class TestMeasurementUnit(unittest.TestCase):
-    def test_unit_equivalence(self):
-        self.assertTrue(MeasurementUnit.create('mm') == MeasurementUnit.create('millimetre'))
-        self.assertTrue(MeasurementUnit.create('Mm') == MeasurementUnit.create('megametre'))
-        self.assertTrue(MeasurementUnit.create('mm') != MeasurementUnit.create('Mm'))
-        self.assertTrue(MeasurementUnit.create('d') == MeasurementUnit.create('day'))
-        self.assertTrue(MeasurementUnit.create('min') == MeasurementUnit.create('minutes'))
-        self.assertTrue(MeasurementUnit.create('%') != MeasurementUnit.create('K'))
 
+    def test_unit_equivalence(self):
+        self.assertTrue(MeasurementUnit.create('mm') ==
+                        MeasurementUnit.create('millimetre'))
+        self.assertTrue(MeasurementUnit.create('Mm') ==
+                        MeasurementUnit.create('megametre'))
+        self.assertTrue(MeasurementUnit.create('mm') !=
+                        MeasurementUnit.create('Mm'))
+        self.assertTrue(MeasurementUnit.create('d') ==
+                        MeasurementUnit.create('day'))
+        self.assertTrue(MeasurementUnit.create('min') ==
+                        MeasurementUnit.create('minutes'))
+        self.assertTrue(MeasurementUnit.create(
+            '%') != MeasurementUnit.create('K'))
 
     def test_canon_conversion(self):
         self.assertTrue(MeasurementUnit.create('cm').to_canon(1) == 10)
         self.assertTrue(MeasurementUnit.create('m').to_canon(1) == 1000)
         self.assertTrue(MeasurementUnit.create('F').from_canon(0) == 32)
-        self.assertTrue(MeasurementUnit.create('microns').to_canon(1800) == 1.8)
+        self.assertTrue(MeasurementUnit.create(
+            'microns').to_canon(1800) == 1.8)
```

Comment:
This can't seriously be recommended style, can it?  Nobody's working on 80-character teletype displays any more, and wrapping makes it way harder to tell that this is parallel to the other tests.  Not to mention that the new block is two levels deep, then almost immediately closes, then continues for some time at one level deep.